easybuild/easyconfigs/g/GLIMMER/GLIMMER-3.02b-ictce-5.3.0.eb

```diff
@@ -20,13 +21,27 @@ toolchain = {'name': 'ictce', 'version': '5.3.0'}
 source_urls = ['http://www.cbcb.umd.edu/software/glimmer']
 sources = ['%%(namelower)s%s.tar.gz' % ''.join(version.split('.'))]
 
+dependencies = [('ELPH', '1.0.1')]
+
 buildopts = '-C ./src'
 
 files_to_copy = ["bin", "docs", "glim302notes.pdf", "lib", "LICENSE", "sample-run", "scripts"]
 
+# add scripts dir to PATH
+modextrapaths = {
+	'PATH': 'scripts'
+}
+
 sanity_check_paths = {
     'files': ["bin/glimmer3", "glim302notes.pdf", "LICENSE"],
     'dirs': ["docs", "lib", "sample-run", "scripts"]
 }
 
+# fix hardcoded path to scripts folder
+postinstallcmds = ["sed -i -e 's|/fs/szgenefinding/Glimmer3/scripts|%(installdir)s/scripts|' %(installdir)s/scripts/*"]
+# fix hardcoded path to bin folder
+postinstallcmds += ["sed -i -e 's|/fs/szgenefinding/Glimmer3/bin|%(installdir)s/bin|' %(installdir)s/scripts/*"]
+# fix hardcoded path to elph binary
+postinstallcmds += ["sed -i -e 's|/nfshomes/adelcher/bin/elph|${EBROOTELPH}/bin/elph|' %(installdir)s/scripts/*"]
```

Comment:
not a big fan of this, but I don't have a really good alternative, at least not until https://github.com/hpcugent/easybuild-framework/issues/1400 is fixed